addOns/soap/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Fix exception with Java 9+ (Issue 4037).
 - SOAP operations are no longer overwritten in sites tree (Issue 1867).
 - Persist the add-on configuration required by the scan rules in the ZAP database (Issue 4866).
+- Fix detection of WSDL files (Issue 6440).
```

Comment:
This needs to be added to the unreleased section.

Reply:
Done.